gigascience/gigadb-website
# Enhances input text color contrast

# Pull request for issue: #2034

This is a pull request for the following functionalities:

* Updated text color for all inputs to increase contrast. Now it's black text on white background

## How to test?

navigate to e.g. http://gigadb.gigasciencejournal.com/adminDataset/update/id/5 and verify input text is black

## How have functionalities been implemented?

there is a class that controls input text color, which I updated, i.e `input:not([type="submit"]):not([type="button"]), select, textarea, .form-control`, this will update the text color of all inputs, select and textareas except for any buttons

## Any issues with implementation?

--

## Any changes to automated tests?

--
